README.md

```diff
@@ -223,6 +220,30 @@ If you get the error that `statsmodels` is not found, you would run
 
 Then please submit an issue so that we can address why that was not imported.
 
+Please note that the package `pmdarima` needs to installed through `pip install` and not through `conda install`.
+
+### Update Terminal
+
+The terminal is constantly being updated with new features and bug fixes, hence, for your terminal to be updaate, you can run:
+```
+git pull
```

Comment:
spelling on updaate

Reply:
also should we specify that you need to be in the right directory?

Reply:
Good point, I'll do that. Thanks!

Reply:
Regarding the directory, I think you can pull from anywhere, even if you are in sub-folders.